resources/views/annotations/show/annotationCanvas.blade.php

```diff
@@ -44,18 +43,18 @@
                     @mixin('imageAnnotationPolygonTools')
                 </control-button>
             </div>
-            <div class="btn-group edit-controls">
-                <control-button icon="fa-tag" title="Attach the currently selected label to existing annotations 𝗟" :active="isAttaching" v-on:click="toggleAttaching">
+            <div class="btn-group edit-controls" v-if="canModify || canDelete">
```

Comment:
The `v-cloak` can also be added here so you don't need it on all of the children.